src/o365/graph/commands/teams/teams-clone.ts

```diff
@@ -0,0 +1,187 @@
+import * as request from 'request-promise-native';
+import auth from '../../GraphAuth';
+import Utils from '../../../../Utils';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import { CommandOption, CommandValidate } from '../../../../Command';
+import GraphCommand from '../../GraphCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  teamId: string;
+  displayName: string;
+  mailNickname: string;
+  partsToClone: string;
+  description?: string;
+  classification?: string;
+  visibility?: string;
+}
+
+class GraphTeamsCloneCommand extends GraphCommand {
+  public get name(): string {
+    return `${commands.TEAMS_CLONE}`;
+  }
+
+  public get description(): string {
+    return 'Creates a copy of a Microsoft Teams team';
```

Comment:
Would it be more precise if we say `Creates a clone of a Microsoft Teams team`?

Reply:
Sure, This is complete. Please verify the latest code.